airbyte-config/specs/build.gradle

```diff
@@ -44,6 +44,7 @@ task generateCombinedConnectorCatalog(type: JavaExec, dependsOn: generateSeedCon
     args 'oss_catalog.json'
 }
 
+// TODO (ben): we need to ensure this is called separately in deploy-oss-catalog.yml so as not to overwrite the catalog downloaded by cloud
```

Comment:
@airbytehq/connector-operations Checking assumptions here

 Am I right in thinking that on the move of platform into platform-internal the below line which triggers `generateCombinedConnectorCatalog` when `processResources` is run should be removed? and then referenced explicitly in the deploy-oss-catalog?

Reply:
Yes. We should not be generating connector catalogs on the platform side. Cloud currently does a similar thing: the catalog generation is its own task that is called directly and has nothing to do with the build.